### ISSUE TITLE
robot verify does not use the catalog-xml

### ISSUE THREAD

**Issue by cmungall** [ontodev/robot#195]
example:

```
~/repos/robot/robot-core/src/test/resources(master) $ robot verify -i import_test.owl --queries test.sparql -O reports/
```

where test.sparql is `SELECT * WHERE { ?x ?r ?y }`

fails with:

```
org.semanticweb.owlapi.model.UnloadableImportException: Could not load imported ontology: <https://github.com/ontodev/robot/robot-core/src/test/resources/simple.owl> Cause: https://github.com/ontodev/robot/robot-core/src/test/resources/simple.owl
```

This works as expected:

```
robot convert -i import_test.owl -o foo.obo
```

This is a bit of a blocker and additionally we have no way of passing in an alternate catalog: 
#41

**Comment by jamesaoverton** [comment]
Use `CommandLineHelper.getInputOntology`, following some of the other commands I’ve written. This should use `IOHelper` which should use the catalog.

I can do it, but not until next week.

**Comment by jamesaoverton** [comment]
In #196 I changed `VerifyCommand` to work like other commands, including catalogs and chaining. 

The bad news is that this didn't solve the problem, because `QueryOperation` only loads the ontology into the graph, not the imports closure. I'm not 100% certain what the right thing to do is.

The good news is that you can now merge then verify, so this should work:

```
robot merge -i import_test.owl verify --queries test.sparql -O reports/
```

Is merge-then-verify acceptable, or do we want to change `QueryOperation`?

**Comment by cmungall** [comment]
I think this is acceptable just now.

I would like the pre-merge state to be such that each imported ontology is in its own NG, but we can leave this as it's own separate ticket #158
